### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by swehba** [dobrovolsky/obsidian-structure#6]
Your plugin sounds intriguing, but how exactly do you use it? Can you provide a simple example?

Do you have to organize notes in a hierarchy in the file system and then navigate? Or, do you somehow specify the parent/child relationships within a note and navigate from there? Really now sure. Thanks.

**Comment by dobrovolsky** [comment]
Created some examples https://github.com/dobrovolsky/obsidain-structure#how-to-organize-your-workspace

**Comment by menagerie198** [comment]
That helps a little, thankyou. This looks a very interesting plugin. I have an issue though. 'Create New Note' only returns an error 'Parent Note does Not exist'. I have tried creating a parent note in the File Explorer - no change. I have tried adding '.sidenote' to an existing note - no luck. The only way that I can get to work is by duplicating and 'renaming' an existing note = '2021-12-03' becomes '2021-12-03.sidenote'. This works, and I can then call the 'parent' note into an adjacent window.
This is very promising, but a bit lumpy. Is this expected behaviour, or am I missing something?
